lib/cisco_node_utils/cmd_ref/bridge_domain_vni.yaml

```diff
@@ -0,0 +1,28 @@
+# bridge_domain
+# bridge_domain feature is available only on n7k
+---
+_exclude: [N3k, N5k, N6k, N9k, ios_xr]
+
+create:
+  set_value: "bridge-domain <crbd>"
+
+member_vni:
+  get_command: "show running-config bridge-domain | inc 'member vni'"
+  get_value: '/^  member vni (.*)/'
+  set_value: "<vnistate> vni <vni> ; bridge-domain <bd> ; <membstate> member vni <membvni>"
```

Comment:
Confused.  `<vnistate> vni <vni>` is an independent command from the others, isn't it?

I expect the config to be something like:

```
n7k8(config)# vni 4100 ; bridge-dom 41 ; member vni 4100
n7k8(config-bdomain)# sh run | sec brid|vni
system bridge-domain 41 
vni 4100
bridge-domain 41
  member vni 4100
```

I suggest you create a separate top-level `vni:` entry instead. It just means you call `config_set` twice when setting these values.

```
member_vni:
  get_value: '/^member vni (\d+)/'
  set_value: "<state> member vni <vni>"

vni:
  get_value: '/^vni (\d+)/'
  set_value: "<state> vni <vni> "
```